app/decorators/manageiq/providers/kubernetes/container_manager_decorator.rb

```diff
@@ -1,7 +1,22 @@
 module ManageIQ::Providers::Kubernetes
-  class ContainerManagerDecorator < MiqDecorator
+  class ContainerManagerDecorator < ExtManagementSystemDecorator
     def self.fileicon
       "svg/vendor-kubernetes.svg"
     end
+
+    def quadicon(_n = nil)
+      {
+        :top_left     => { :text => container_nodes.size },
+        :top_right    => { :text => enabled? ? "on" : "paused" },
```

Comment:
Where is the on/paused taken from? I don't see the original code from before the refactoring, so I wonder...

Reply:
This comes from [app/helpers/quadicon_helper.rb:571](https://github.com/ManageIQ/manageiq-ui-classic/pull/3046/files#diff-6525cb9600835fcc2ba062fde39a2292L571). However this is incorrect because it should be icon and not text.